src/api/api.js

```diff
@@ -83,21 +83,14 @@ const getParcelInfo = async(id) => {
 
 const getParcelContent = async(docid) => {
 
-  /* TEST 
-  let parcelContent = {
-    name: "Dal Lake",
-    webContent: "https://www.beautifulworld.com/asia/india/dal-lake/",
-    mediaContent: [
-      {"name": "Lake Boat", "contentUrl" : "QmSJZ2DvrLz2hBeduYiPTsSmCXViZrbj1cc2V3mud8kX6N"},
-      {"name": "Mountain Bike", "contentUrl" : "bafybeifvwcgfg6he3ihovvhakrdlihh5ol4rucu26ou5zel3jlhgo4acay"}
-    ]
+  let doc = null;
+  
+  try{
+    doc = await ceramic.loadStream(docid);
```

Comment:
See the new note here in Ceramic docs: 
- https://developers.ceramic.network/build/queries/#query-a-stream
- https://developers.ceramic.network/streamtypes/tile-document/api/#load-a-document

There is a new API specifically for Tile documents `TileDocument.load`. I'm not sure if you need to use it or not. It seems like it is only needed for making changes?